overlord/snapstate/snapmgr.go

```diff
@@ -489,6 +498,7 @@ func (m *SnapManager) NextRefresh() time.Time {
 // RefreshSchedule returns the current refresh schedule.
 // The caller should be holding the state lock.
 func (m *SnapManager) RefreshSchedule() string {
+	m.checkRefreshSchedule()
```

Comment:
are we sure we can ignore errors here?  anyway I think the one place using this could survive this returning an error, it's not used all over the place